history/titles/00_k_hyjal.txt

```diff
@@ -45,9 +45,12 @@ c_lardonir = {
 	25.2.23={
 		holder=340021 #Maiev Shadowsong
 	}
-	604.10.12={
+	615.1.30={
 		holder=340101 #Sira Moonwarden
 	}
+	616.1.21={
+		liege=e_horde # Sira is raised as undead by Sylvanas
+	}
```

Comment:
`d_wardens` is the liege. If `d_wardens` is a vassal of the Horde, it creates a hierarchy up to this county.

Reply:
Fair, but I removed this in a73fedcf459b5b9f6d7920599cfffd5b7949cfe3 anyway since I made Maiev the leader instead of making `e_horde` their liege.